include/mesh/mesh_function.h

```diff
@@ -159,6 +168,25 @@ class MeshFunction : public FunctionBase<Number>,
                    const std::set<subdomain_id_type> * subdomain_ids);
 
   /**
+   * Similar to operator() with the same parameter list, but with the difference
+   * that multiple values on faces are explicitly permitted. This is useful for
+   * discontinuous shape functions that are evaluated on faces.
+   */
+  void discontinuous_value (const Point & p,
+                            const Real time,
+                            std::map<const Elem *, DenseVector<Number> > & output);
```

Comment:
What's the reasoning behind using `DenseVector` here?  I would prefer just using `std::vector`.